core/app/components/FormBuilder/ElementPanel/InputComponentConfigurationForm.tsx

```diff
@@ -53,6 +54,17 @@ const DatePicker = dynamic(() => import("ui/date-picker").then((mod) => mod.Date
 	loading: () => <Skeleton className="h-9 w-full" />,
 });
 
+const ContextEditorClient = dynamic(
+	() =>
+		import("~/app/components/ContextEditor/ContextEditorClient").then(
+			(mod) => mod.ContextEditorClient
+		),
+	{
+		ssr: false,
+		loading: () => <Skeleton className="h-9 w-full" />,
+	}
+);
+
```

Comment:
this was not being lazy loaded for some reason, which slows down the initial loading of the form page a lot